pkg/bundle/manager.go

```diff
@@ -23,14 +25,16 @@ type bundleManager struct {
 	bundleClient   Client
 	registryClient RegistryClient
 	targetClient   authenticator.TargetClusterClient
+	config         config.Config
 }
 
-func NewBundleManager(log logr.Logger, registryClient RegistryClient, bundleClient Client, targetClient authenticator.TargetClusterClient) *bundleManager {
+func NewBundleManager(log logr.Logger, registryClient RegistryClient, bundleClient Client, targetClient authenticator.TargetClusterClient, config config.Config) *bundleManager {
 	return &bundleManager{
 		log:            log,
 		bundleClient:   bundleClient,
 		registryClient: registryClient,
 		targetClient:   targetClient,
+		config:         config,
```

Comment:
This config is such a generic name, I had no idea what it was until I got further down.